Description: remove the target dependency in order to avoid unnecessary rebuild
 that would incur FTBFS.
Author: Mo Zhou
Last-Update: 2019-08-28
Index: lapack/BLAS/Makefile
===================================================================
--- lapack.orig/BLAS/Makefile
+++ lapack/BLAS/Makefile
@@ -9,7 +9,7 @@ blas:
 	$(MAKE) -C SRC
 
 .PHONY: blas_testing
-blas_testing: blas
+blas_testing:
 	$(MAKE) -C TESTING run
 
 .PHONY: clean cleanobj cleanlib cleanexe cleantest
